core/sql/cli/CliExtern.cpp

```diff
@@ -6316,7 +6316,8 @@ Lng32 SQL_EXEC_DeleteHbaseJNI()
       threadContext->incrNumOfCliCalls();
 
       HBaseClient_JNI::deleteInstance();
-      HiveClient_JNI::deleteInstance();
+      // The Hive client persists across connections
+      // HiveClient_JNI::deleteInstance();
```

Comment:
I would echo Selva's comment. Can you please explain how the HiveClient_JNI object is different from HBaseClient_JNI? I had tried to model HiveClient_JNI after HBaseClient_JNI and would like check if that assumption is incorrect in other places too.